app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/event_screen/EventViewModel.kt

```diff
@@ -171,7 +188,18 @@ class EventViewModel @Inject constructor(
                 _state.update { _state ->
                     _state.copy(
                         editMode = null,
-                        addAttendeeDialogErrorMessage = null,
+                    )
+                }
+                sendEvent(ClearErrorsForAddAttendeeDialog)
+            }
+            is ValidateAttendeeEmail -> {
+                _state.update { _state ->
+                    _state.copy(
+                        isAttendeeEmailValid =
+                            if (uiEvent.email.isBlank())
+                                    null
+                                else
+                                    validateEmail.validate(uiEvent.email)
                     )
```

Comment:
Ok seems like it is null, why not make it `false` by default, since an empty string is not a valid email?

Reply:
Because I want to start the user out with a "clean, error-free" dialog state. And if its empty, there is no error yet. Only when the user starts typing do I indicate the validity of the text as an email address.